style.css

```diff
@@ -0,0 +1,535 @@
+/* variables */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+:root {
+  --mainC: #6070ff;
+  --secC: #344563;
+  --sec2C: #7f8cff;
+  --mainF: 'Poppins', cursive;
+}
+
+/* mobile styles */
+* {
+  margin: 0;
+  font-family: var(--mainF);
+}
+
+body {
+  min-height: 100vh;
+  background: #f7f7f9;
+}
+
+ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+li {
+  padding: 10px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px 30px;
+  background: white;
+  position: fixed;
+  top: 0;
+  width: calc(100% - 60px);
+}
+
+.logo {
+  color: var(--mainC);
+  font-weight: 700;
+  font-size: 18px;
+  padding: 10px;
+}
+
+.pointer {
+  cursor: pointer;
+}
+
+.pointer:hover {
+  color: #6070ff;
+}
+
+#desk-list {
+  display: none;
+}
+
+#Union-menu {
+  display: unset;
+  padding: 18px;
+}
+
+main {
+  border-bottom-left-radius: 100px;
+  margin-bottom: 100px;
+  padding: 100px 40px 80px 40px;
+  background: white url('./images/Header.svg') no-repeat;
+  background-size: contain;
+  margin-top: 75px;
+}
+
+h1 {
+  margin-right: 80px;
+  font-weight: 700;
+  line-height: 52px;
+  font-family: var(--secC);
+}
+
+p {
+  margin: 10px 0;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 28px;
+  color: #344563;
+  font-family: var(--mainF);
+  font-style: normal;
+  text-align: justify;
+  text-justify: inter-word;
+}
+
+a {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--sec2C);
+}
+
+.socials {
+  width: 180px;
+  padding: 0;
+  margin-top: 1.625em;
+  margin-bottom: 2.125em;
+}
+
+.socials :nth-child(1) {
+  padding-left: 0;
+}
+
+#projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 142px 142px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 330px;
+  background: #fff;
+  border: 2px solid #dfe1e6;
+  border-radius: 16px;
+  box-shadow: 0 0 0.5px 0.25px inset #c8c6c6;
+  padding: 16px 16px 30px 16px;
+  margin-bottom: 90px;
+  transition: all ease-in-out 0.2s;
+}
```

Comment:
- [ ] Good job using the `FlexBox` for project cards. Please use the `CSS grid` for the project's main container as requested in the project requirements.
![image](https://user-images.githubusercontent.com/83803414/187044801-324bd5fb-3e83-4bb5-873e-eaabd0d7e483.png)